src/wallet/wallet.h

```diff
@@ -658,6 +642,26 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+struct CoinSelectionParams
+{
+    bool use_bnb;
+    size_t change_output_size;
+    size_t change_spend_size;
+    CFeeRate effective_fee;
+    size_t tx_noinputs_size;
+
+    CoinSelectionParams(bool use_bnb, size_t change_output_size, size_t change_spend_size, CFeeRate effective_fee, size_t tx_noinputs_size) : use_bnb(use_bnb), change_output_size(change_output_size), change_spend_size(change_spend_size), effective_fee(effective_fee), tx_noinputs_size(tx_noinputs_size) {}
+    CoinSelectionParams() : use_bnb(true), change_output_size(0), change_spend_size(0), effective_fee(CFeeRate(0)), tx_noinputs_size(0) {}
```

Comment:
nit: less boiler plate if you use a delegating ctor

Reply:
How does that work?

Reply:
> How does that work?

He was suggesting you write:

```c++
CoinSelectionParams() : CoinSelectionParams(true, 0, ...) {}
```

IMO, it would actually be better to use member initialization and write:

```
bool use_bnb = true;
size_t change_output_size = 0;
...
CoinSelectionParams(bool use_bnb, size_t change_output_size, ...) ...;
CoinSelectionParams() {}
```

Reply:
I've used @ryanofsky's suggestion